stable/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -80,7 +80,7 @@ spec:
 {{- end }}
         livenessProbe:
           exec:
-            command: [ "sh", "/probes/check-quorum.sh", "{{ .Values.redis.port }}"]
+            command: [ "sh", "/probes/readiness.sh", "{{ .Values.redis.port }}"]
```

Comment:
Maybe a good idea to disable it on the sentinel probe as well.

Reply:
Done